modules/expo-permissions/ios/EXPermissions/EXLocationRequester.m

```diff
@@ -16,15 +20,20 @@ @interface EXLocationRequester () <CLLocationManagerDelegate>
 
 @implementation EXLocationRequester
 
++ (void)load
+{
+  alwaysAuthorizationSelector = NSSelectorFromString([@"request" stringByAppendingString:@"AlwaysAuthorization"]);
+  whenInUseAuthorizationSelector = NSSelectorFromString([@"request" stringByAppendingString:@"WhenInUseAuthorization"]);
+}
+
 + (NSDictionary *)permissions
 {
   EXPermissionStatus status;
   NSString *scope = @"none";
   
   CLAuthorizationStatus systemStatus;
-  NSString *whenInUseUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationWhenInUseUsageDescription"];
-  if (!whenInUseUsageDescription) {
-    EXFatal(EXErrorWithMessage(@"This app is missing NSLocationWhenInUseUsageDescription, so location services will fail. Add one of these keys to your bundle's Info.plist."));
+  if (![self isConfiguredForAlwaysAuthorization] && ![self isConfiguredForWhenInUseAuthorization]) {
+    EXFatal(EXErrorWithMessage(@"This app is missing usage descriptions, so location services will fail. Add one of the `NSLocation*UsageDescription` keys to your bundle's Info.plist."));
```

Comment:
It seems to me that we should be more clear here. In fact, they must be added to `app.json` 🤔